.fmt/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
@subpop Note that this file is incorrect, it should be `.fmf/version`

To mitigate the issue, you can use next time `tmt init` that creates it correctly